app/views/external_links/_all.html.erb

```diff
@@ -1,5 +1,8 @@
-<%= render partial: 'external_links/psu_digital_collections_links', locals: { psu_digital_collections_links: document.psu_digital_collections_links } if document.psu_digital_collections_links %>
+<% psu_digital_collections_links = document.psu_digital_collections_links
+   access_online_links = document.access_online_links %>
```

Comment:
Do we want to add `hathi_links` to this list of variables, and refactor the partial declaration similarly?

Reply:
yeah good idea 👍

Reply:
this is done with the latest commit, thanks 👍